claragenomics/dl4atac/models/model_utils.py

```diff
@@ -41,6 +41,25 @@
 _logger.addHandler(_handler)
 
 
+def type_or_none_fn(type):
+    """Ensure provided arguments are not defined at the same time.
```

Comment:
this description doesn't seem to be right. it should be

```
Generate function to interpret a string as a specific type or None
```

Reply:
Yes, good catch. @avantikalal  please correct this in cmd_args.py as well.

Reply:
Fixed in both.

Reply:
Thanks!